.github/workflows/publish.yml

```
@@ -0,0 +1,28 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.x'
```

Let's cache pip downloads to help reduce the load on PyPI:

https://dustingram.com/articles/2021/04/14/powering-the-python-package-index-in-2021/

```suggestion
      uses: actions/setup-python@v4
      with:
        python-version: '3.x'
        cache: pip
        cache-dependency-path: setup.py
```